gapic/src/main/com/google/gapid/Server.java

```diff
@@ -121,9 +119,9 @@ private void fetchServerInfo()
     Service.ServerInfo info = Rpc.get(client.getSeverInfo(), FETCH_INFO_TIMEOUT_MS, MILLISECONDS);
     LOG.log(INFO, "Server info: {0}", info);
     Version gapisVersion = Version.fromProto(info);
-    if (!GAPIC_VERSION.isCompatible(gapisVersion)) {
+    if (!Version.GAPID_VERSION.isCompatible(gapisVersion)) {
```

Comment:
Leaving it as a static import would mean you don't have the "Version" part twice..